lib/output/output.go

```diff
@@ -1,11 +1,8 @@
 package output
 
-import (
-	"encoding/json"
-	"fmt"
-	"sort"
-	"strings"
-)
+type OutputBuilder interface {
```

Comment:
I believe implementations have to match exactly the same number of methods as well as exact input and output types.

In this case, if a struct would implement anything other than GenerateOutput, it would no longer conform to the interface.